smarts/core/sensors.py

```diff
@@ -458,15 +458,16 @@ def _is_done_with_events(cls, sim, agent_id, vehicle, sensor_state):
         )
 
         done = (
-            (is_off_road and done_criteria.off_road)
+            (not sim._resetting) and 
```

Comment:
It would  be better to use a non-private property here, i.e., to promote `_resetting` to `resetting` in the Smarts class.